roles/restore/tasks/deploy_awx.yml

```diff
@@ -1,27 +1,5 @@
 ---
 
-- name: Get AWX object definition from pvc
```

Comment:
These tasks were moved to a new `import_vars.yml` file, which is now called before `secrets.yml` and `awx-cro.yml`.  This is because this `previous_deployment_name` variable needs to be available for use in the secrets.yml file.